index.html

```diff
@@ -2092,319 +1905,526 @@ <h1>Core Representations</h1>
+    "id": "did:example:123456789abcdefghi#photos",
+    "type": "PhotoStreamService",
+    "serviceEndpoint": "https://example.org/photos/379283"
+  }</span>]
+}
+        </pre>
 
-    <section>
-        <h2>
-JSON-LD
-        </h2>
+        <p>
+The examples so far show that it is easy to extend the
+<a>decentralized identifiers</a> data model in a permissionless and
+decentralized way. The mechanism also ensures that
+<a>decentralized identifiers</a> created in this way prevent namespace conflicts
+and semantic ambiguity.
+        </p>
 
         <p>
-[[!JSON-LD]] is a JSON-based format used to serialize
-<a href="http://www.w3.org/TR/ld-glossary/#linked-data">Linked Data</a>.
+An extensibility model this dynamic does increase the implementation burden.
+Software written for such a system needs to determine if accepting
+<a>DID documents</a> with extensions is acceptable based on the risk profile of
+the application. Some applications might choose to accept but ignore extensions,
+others might choose to only accept certain extensions, while highly secure
+environments might disallow extensions. These decisions are up to the
+application developers and are specifically not the domain of this
+specification.
         </p>
 
         <p>
-When producing and consuming DID documents that are in JSON-LD (as indicated by
-a <code>content-type</code> of <code>application/did+ld+json</code> in the
-resolver metadata), the following rules MUST be followed.
+Implementations MUST produce an error when an extension JSON-LD Context
+overrides the expanded URL for a term specified in this specification. To avoid
+the possibility of accidentally overriding terms, developers SHOULD scope their
+extensions. For example, the following extension scopes the new
+<code>PhotoStreamService</code> term so that it can only be used within the
+<code>service</code> property:
         </p>
 
-        <ul>
-            <li>
-The <code>@id</code> and <code>@type</code> keywords are aliased to
-<code>id</code> and <code>type</code> respectively, enabling developers to use
-this specification as idiomatic JSON.
-            </li>
-            <li>
-Even though JSON-LD allows any IRI as node identifiers, <a>DID documents</a>
-are explicitly restricted to only describe DIDs. This means that the value of
-<code>id</code> MUST be a valid <a>DID</a> and not any other kind of IRI.
-            </li>
-            <li>
-Data types, such as integers, dates, units of measure, and URLs, are
-automatically typed to provide stronger type guarantees for use cases that
-require them.
-            </li>
-        </ul>
+        <pre class="example nohighlight" title="Scoping terms in a JSON-LD Context">
+{
+  "@context": {
+    <span class="highlight">"service": {
+      "@id": "https://w3id.org/did#service",
+      "@context": {
+        "PhotoStreamService": "https://example.com/vocab#PhotoStreamService"
+      }
+    }</span>
+  }
+}
+        </pre>
+
+    <section>
+      <h2>CBOR</h2>
```

Comment:
not sure why there is so much of a diff from master upstream branch.  but CBOR section starts here.

Reply:
It feels like your PR is based on some older version of the document. Can you rebase or recreate the PR from the latest master, so that nothing is changed except for the CBOR section itself?

Reply:
Ok, this looks better.  Old version, needed to rebase.